test/groovy/templates/PiperPipelineStageAcceptanceTest.groovy

```diff
@@ -0,0 +1,42 @@
+#!groovy
+package templates
+
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import util.*
```

Comment:
[minor] we should IMO use individual imports since this avoids misunderstanding, especially when reading code outside an IDE, e.g. github web ui. Applies also elsewhere